openpathsampling/tests/test_helpers.py

```diff
@@ -232,4 +239,41 @@ def current_snapshot(self, snapshot):
 
     def generate_next_frame(self):
         self._current_snapshot = None
-        return self.current_snapshot
\ No newline at end of file
+        return self.current_snapshot
+
+
+class IdentityPathMover(paths.PathMover):
```

Comment:
`IdentityPathMover` might move to the main `pathmover.py`. This is the no-op move I've talked about in #349, so we might use it there.